src/services/api_key_service.js

```diff
@@ -0,0 +1,8 @@
+import baseService from './base_service';
+
+export default (easypostClient) =>
+  class ApiKeyService extends baseService(easypostClient) {
```

Comment:
"create", "all" and "retrieve" have been inherited from the base class and not overridden. That would mean there is a function to "create an API key", which is incorrect (see my comment on the base service re: inherited functions)

Reply:
This may be an outlier, most should state something isn't implemented. I'll add that here. 

Were there others you noticed?